react/w-books/src/app/components/BookDetail/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Link, useHistory } from 'react-router-dom';
+import i18next from 'i18next';
+
+import { getBookById } from '~services/booksService';
+
+import { useRequest } from '../../hooks/useRequest';
+import badge from '../../assets/badge.png';
+
+import styles from './styles.module.scss';
+
+function BookDetail() {
+  const history = useHistory();
+  const { pathname } = history.location;
+  const bookId = pathname.split('/')[pathname.split('/').length - 1];
+  const [state]: any = useRequest({ request: getBookById, payload: bookId }, []);
+  const book = state ? state.data : false;
+
+  return (
+    <div className={styles.homeContainer}>
+      <div className={styles.backButton}>
+        <Link to="/home">
+          <div className={styles.arrow}>
+            <span className={styles.chevronLeft}>&lt;</span>
+            <div>{i18next.t('BookDetail:goBack')}</div>
+          </div>
+        </Link>
+      </div>
+
+      {book && (
+        <div className={styles.bookDetailContainer}>
+          <img className={styles.badge} src={badge} alt="Badge" />
```

Comment:
Remember to use as less html code as possible for clarity and reusability. In this case, try just to use just the container element and apply the image as a decorator using only CSS (hint: you can use the ::before selector), that way you will have a single reusable html tag (the container) that contains the  decorative imanage via CSS only. If you have any doubt about this ask me.